apps/web/src/components/Publication/Type/Collected.tsx

```diff
@@ -16,13 +16,7 @@ const Collected: FC<Props> = ({ publication }) => {
     <div className="flex items-center pb-4 space-x-1 lt-text-gray-500 text-[13px]">
       <CollectionIcon className="w-4 h-4" />
       {publication?.collectedBy?.defaultProfile ? (
-        <Link href={`/u/${formatHandle(publication?.collectedBy?.defaultProfile?.handle)}`}>
-          {publication?.collectedBy?.defaultProfile?.name ? (
-            <b>{publication?.collectedBy?.defaultProfile?.name}</b>
-          ) : (
-            <Slug slug={formatHandle(publication?.collectedBy?.defaultProfile?.handle)} prefix="@" />
-          )}
-        </Link>
+        <UserName profile={publication.profile} className="max-w-xs truncate" />
```

Comment:
Is this correct? it should be inside `collectedBy` right?

Reply:
Ohh, no it's not correct, good catch! I have updated it now.